README.md

```diff
@@ -1,51 +1,99 @@
-# Mini Pupper ROS2
+# Mini Pupper ROS 2
 
 ## 1. Installation
-Ubuntu 22.04 + ROS2 Humble is required. Please follow the [installation document for ROS Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)
+
+* To use Gazebo simulator, "1.1 PC Setup" is required.
+* To control Mini Pupper, "1.2 Mini Pupper Setup" is required.
+* To control Mini Pupper using visualize tools, "1.1 PC Setup" and "1.2 Mini Pupper Setup" is required.
+
+### 1.1 PC Setup
+
+PC Setup corresponds to PC (your desktop or laptop PC) for controlling Mini Pupper remotely or execute simulator.  
+__Do not apply these PC Setup commands to your Raspberry Pi on Mini Pupper.__
+
+Ubuntu 22.04 + ROS 2 Humble is required.  
+Please follow the [installation document for ROS Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html) or use the [unofficial ROS 2 installation script](https://github.com/Tiryoh/ros2_setup_scripts_ubuntu).
+
+After ROS 2 installation, download the Mini Pupper ROS package in the workspace.
 
 ```sh
-cd ~
-mkdir -p ros2_ws/src
-cd ros2_ws/src
+mkdir -p ~/ros2_ws/src
+cd ~/ros2_ws/src
 git clone https://github.com/mangdangroboticsclub/mini_pupper_ros.git -b ros2
-git clone --recursive https://github.com/chvmp/champ -b ros2
-cd ..
+vcs import < minipupper_ros/.minipupper.repos --recursive
+```
+
+Build and install all ROS packages.
+
+```sh
+cd ~/ros2_ws
 rosdep install --from-paths src --ignore-src -r -y
+sudo apt-get install ros-humble-teleop-twist-keyboard
 colcon build --symlink-install
+```
+
+### 1.2 Mini Pupper Setup
```

Comment:
It's probably worth mentioning the preconditions related to MiniPupper drivers setup. Seems like `*_bsp` repos weren't updated for a while. Anyway, it'd make sense to add a reference to the actual setup instructions.

Reply:
Thanks, added. [ac192f5](https://github.com/mangdangroboticsclub/mini_pupper_ros/pull/40/commits/ac192f577059512f5ed89231aacca29b919f54f2)
The latest bsp package is https://github.com/mangdangroboticsclub/mini_pupper_bsp, not minipupper_ros_bsp